examples/sending/suppressions.ts

```diff
@@ -0,0 +1,30 @@
+import { MailtrapClient } from "mailtrap";
+
+const TOKEN = "<YOUR-TOKEN-HERE>";
+const ACCOUNT_ID = "<YOUR-ACCOUNT-ID-HERE>";
+
+const client = new MailtrapClient({
+  token: TOKEN,
+  accountId: ACCOUNT_ID
+});
+
+async function suppressionsFlow() {
+  // Get all suppressions
+  const allSuppressions = await client.suppressions.getList();
+  console.log("All suppressions:", allSuppressions);
+
+  // Get suppressions filtered by email
+  const filteredSuppressions = await client.suppressions.getList("test@example.com");
```

Comment:
I'd make this a named parameter, so that it's easier to add more named filters in the future.

```suggestion
  const filteredSuppressions = await client.suppressions.getList({email: "test@example.com"});
```

Reply:
@narekhovhannisyan this example should be updated after we changed the arguments